### DISCUSSION TITLE
Redirect funds from sponsors to sponsor other projects?

### DISCUSSION THREAD

**Discussion by YuriGor** [community/community#22066] [New to GitHub]
<p>Hi, is it possible to distribute incoming funds over other projects / people without withdrawing it first?<br>
Imagine there is a core team in the project and I want to share.</p>
<p>Or if the project depends on some key libraries that also need sponsor help.</p>
<p>If I will payout everything and then spend it back for sponsorship</p>
<ul>
<li>all the taxes, currency conversions, and other fees will be applied two or three times…</li>
</ul>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome back!</p>
<p>I’m afraid that isn’t an option available at present, but I can see how would be useful. Keep your eyes open for new features though - Sponsors is still very new so you’ll see new things coming up for sure.</p>
